src/borg/repository.py

```diff
@@ -540,7 +540,8 @@ def rename_tmp(file):
         # Log transaction in append-only mode
         if self.append_only:
             with open(os.path.join(self.path, 'transactions'), 'a') as log:
-                print('transaction %d, UTC time %s' % (transaction_id, datetime.utcnow().isoformat()), file=log)
+                print('transaction %d, UTC time %s' % (
+                    transaction_id, datetime.utcnow().strftime(ISO_FORMAT)), file=log)
```

Comment:
Indentation is off-by-one

Reply:
it's indented by 4, but yes, prettier would be to indent to in-parens level.